cmd/edit.go

```diff
@@ -60,39 +63,32 @@ func newEditCmd() *cobra.Command {
 	return cmd
 }
 
-var _ commandOptions = &editOptions{}
+var _ cmdutil.RunOptions = &editOptions{}
 
 type editOptions struct {
+	config *config.Config
+
 	multigroup bool
 }
 
 func (o *editOptions) bindFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&o.multigroup, "multigroup", false, "enable or disable multigroup layout")
 }
 
-func (o *editOptions) loadConfig() (*config.Config, error) {
-	projectConfig, err := config.Load()
-	if os.IsNotExist(err) {
-		return nil, errors.New("unable to find configuration file, project must be initialized")
-	}
-
-	return projectConfig, err
-}
-
-func (o *editOptions) validate(c *config.Config) error {
-	if !c.IsV2() {
-		if c.MultiGroup {
-			return fmt.Errorf("multiple group support can't be enabled for version %s", c.Version)
+func (o *editOptions) Validate() error {
+	if !o.config.IsV2() && !o.config.IsV3() {
```

Comment:
```suggestion
	if !o.config.IsV1() {
```

Reply:
Will make easier remove the if to clean the code if we remove V1

Reply:
Not sure if this was the intent, but using `!o.config.IsV2() && !o.config.IsV3()` means that we would need to explicitly assert that a future version (e.g. v4) supports multi-group. This is probably safer than _assuming_ that all future versions will support multi-group.

Reply:
Once #1355 is merged I will rebase to remove v1 checks.